webpack/assets/javascripts/react_app/common/withReactRoutes.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+import { Router } from 'react-router-dom';
+import history from '../history';
+import AppSwitcher from '../routes';
+
+const withReactRoutes = Component => props => (
```

Comment:
IMO we are going to see more uses of `withReactRoutes`, can you document it in the storybook? I guess MDX style would work better